classes/table/accessaudit_table.php

```diff
@@ -142,6 +142,8 @@ public function query_db($pagesize, $useinitialsbar = true) {
      * @return array containing sql to use and an array of params.
      */
     protected function get_sql_and_params($count = false) {
+        global $CFG;
+
         if ($count) {
             $select = "COUNT(1)";
         } else {
```

Comment:
Split this $select into two parts, a $userselect, and a $roleselect, you can reuse the first in the both sql below,but only the $userselect can be used in the first one

Reply:
Hi @brendanheywood  This is fixed in commit 4de144be78a96d392ba0af40772a179037e29d49